src/types/todo-type.ts

```diff
@@ -0,0 +1,3 @@
+export type ToDoProps = {
+  todo: { id: number; title: string; description: string };
+};
```

Comment:
se esse type fosse apenas assim:
```suggestion
export type ToDoProps = {
  id: number; 
  title: string; 
  description: string
};
```

seria mais facil de vc reaproveitar esse type em varios pontos do projeto